### DISCUSSION TITLE
Polymorphic relations -  recursive relationship retrieving

### DISCUSSION THREAD

**Discussion by SneakyGerald** [laravel/framework#35622] [Q&A]
- Laravel Version: 7.9.2
- PHP Version: 7.2.29
- Database Driver & Version: 10.1.44-MariaDB

### Description:
I'm using Polymorphic relations on my comments table. I can create comments on Post model or replies on Comment model. However, I'm not able to retrieve the relationship of the original comment when I already accessed commetable method on the reply comment to retrieve the original comment relationship.

### Steps To Reproduce:
1. Make comment on Post
2. Reply on that comment
3. Our reply comment in database will have commentable fields pointing to Comment model and the id of the original comment.
4. When we try to extract the relationship of the original comment `$comment->commentable` we will obtain Post releated information.
5. When we try to extract the relationship of the reply comment `$reply->commentable` we will obtain original Comment releated information.
6. Now, when we try to get ones again the relationship of the original comment (reply->original-comment->Post) it doesn't contains comentable relationship:
![image](https://user-images.githubusercontent.com/26321350/102267682-91a2d780-3f1a-11eb-9f63-a3233c665819.png)

As you can see:
- First item of the array is `$reply` (reply with its relationship on Comment to get the original comment)
- Second item of the array is `$reply->commentable` (original comment obtained from reply with missing relationship on Post)
- Now, I want to get Post relationship of that original comment but I'm not able: `$reply->commentable->commentable`

**Note**: Instead of using default commentable method I'm using my own string 'section'.

**Comment by Blair2004** [comment]
Hi, does your issue is related to any specific application? When you mention method "commentable", it looks like it's a relation determined on your model. It might be more helpful to answer if you can provide the relationship you build.
